README.md

```diff
@@ -1,33 +1,164 @@
-To use this template create a new repository and select `example-pkg` from the template dropdown. Make sure to name your repository following `<name>-pkg` convention. Make sure to apply branch protection rules to `main` branch and enable `dependabot` for security updates.
-If you're interested in automated dependency updates make sure that `renovate` has access to the new repository.
-
-# Things to do change when you use this repository as a template:
-- [ ] replace `example-pkg` with the name of your package in this `README`
-- [ ] update the `@smg-automotive/example` with the name of your package in `package.json`
-- [ ] update `repository` and `homepage` sections in `package.json` to point to your GitHub repository
-- [ ] ensure that Frontend team has the admin access to the repository
-- [ ] ensure that the Bots team has write access to the repository (this is needed to release the package)
-- [ ] provide usage examples in the `README.md`
-- [ ] enable the project on circleci.com to build and test your package
-- [ ] change the circleci status badge in this `README` to the new project
-- [ ] develop an awesome package
-- [ ] live long and prosper
-# example-pkg
-
-[![CircleCI](https://circleci.com/gh/smg-automotive/example-pkg/tree/main.svg?style=svg&circle-token=c183f151fea3c74453cf8dd962d31e115906a300)](https://circleci.com/gh/smg-automotive/example-pkg/tree/main)
+# api-client-pkg
+
+[![CircleCI](https://circleci.com/gh/smg-automotive/api-client-pkg/tree/main.svg?style=svg&circle-token=c183f151fea3c74453cf8dd962d31e115906a300)](https://circleci.com/gh/smg-automotive/example-pkg/tree/main)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
+
 ```
-npm install @smg-automotive/example
+npm install @smg-automotive/api-client-pkg
 ```
 
+````typescript
+import { ApiClient } from "@smg-automotive/api-client-pkg"
+````
+
+### Configuration
+
+The available instance methods are listed below. The specified config will be merged with the instance config.
+
+````typescript
+ApiClient.configure({
+  baseUrl: 'https://api.automotive.ch/api',
+  headers: { 'Content-Type': 'application/json' },
```

Comment:
wouldn't that be a sane default to set internally? what's the case where we need to override this?

Reply:
Maybe we can go with the default and allowing to override? We used `Content-Type` to control the API version in the past and I assume this would still be the case in the new platform.

Reply:
it is the default. Let me replace that with a better example